examples/plugin/wasi-crypto/README.md

```diff
@@ -0,0 +1,49 @@
+# WasmEdge WASI-Crypto example.
+
+This is a example for demonstrate how to use wasi-crypto plugin of WasmEdge in Rust which is adopted from wasi-crypto tests.
+
+## Prerequisites
+
+### Install Rust.
+
+Follow the instructions below to install rust and wasm32-wasi target.
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+rustup target add wasm32-wasi
+```
+
+### Install WasmEdge and WASI-crypto plugin.
+
+Note that if you install WasmEdge using install script, you need to download `wasi-crypto` plugin from [release page](https://github.com/WasmEdge/WasmEdge/releases/) and put it into `$HOME/.wasmedge/plugin/`.
+
+Or you can build wasmedge from scratch with wasi-crypto plugin enabled.
+
+```sh
+git clone https://github.com/WasmEdge/WasmEdge.git --depth 1
+cd WasmEdge
+mkdir build; cd build
+cmake .. -DWASMEDGE_PLUGIN_WASI_CRYPTO=ON
+make
+# to tell wasmedge where to find wasi-crypto plugin.
+export WASMEDGE_PLUGIN_PATH=$PWD/plugins/wasi_crypto
+# compiled wasmedge is located in: ./tools/wasmedge/wasmedge
+```
+
+## Build the example
+
+```sh
+cargo b --target wasm32-wasi
+```
+
+Then we get `target/wasm32-wasi/debug/wasi-crypto-example.wasm`.
+
+## Run the example
+
+We can run this example with `wasmedge` like
+
+```sh
+wasmedge target/wasm32-wasi/debug/wasi-crypto-example.wasm
+```
+
+This example should run successfully and print out the signatures.
```

Comment:
Is the output the same every time? If so, please post the execution output for users to check if the result is exactly matched.